platform/core/src/studies/retrieveStudyMetadata.js

```diff
@@ -33,18 +40,75 @@ export function retrieveStudyMetadata(server, StudyInstanceUID, filters) {
   }
 
   // Create a promise to handle the data retrieval
-  const promise = new Promise((resolve, reject) => {
-    RetrieveMetadata(server, StudyInstanceUID, filters).then(function(data) {
-      resolve(data);
-    }, reject);
-  });
+  let promise;
+
+  if (
+    filters &&
+    filters.seriesInstanceUID &&
+    seperateSeriesInstanceUIDFilters
+  ) {
+    promise = _seperateSeriesRequestToAggregatePromise(
+      server,
+      StudyInstanceUID,
+      filters
+    );
+  } else {
+    promise = new Promise((resolve, reject) => {
+      RetrieveMetadata(server, StudyInstanceUID, filters).then(function(data) {
+        resolve(data);
+      }, reject);
+    });
+  }
 
   // Store the promise in cache
   StudyMetaDataPromises.set(StudyInstanceUID, promise);
 
   return promise;
 }
 
+/**
+ * Splits up seriesInstanceUID filters to multiple calls for platforms
```

Comment:
can we capitalize SeriesInstanceUID?

Reply:
I'm on the fence about this, `seriesInstanceUID` is the actual parameter passed to dicomweb client, that is the parameter this splits up. So I think it makes sense to have the jsdoc reflect this.